sos/sosreport.py

```diff
@@ -1005,6 +1005,7 @@ def _collect_plugin(self, plugin):
                 self.ui_log.error("\n Plugin %s timed out\n" % plugin[1])
                 self.running_plugs.remove(plugin[1])
                 self.loaded_plugins[plugin[0]-1][1]._timeout_hit = True
+                self.loaded_plugins[plugin[0]-1][1].stop()
```

Comment:
Should we move
```python
self.loaded_plugins[plugin[0]-1][1]._timeout_hit = True
```

into the new `stop()` method? Or in some other way merge _stopped with _timeout_hit? Perhaps `check_timeout()` should now just `return self._stopped` instead of a separate `_timeout_hit` var?